.github/workflows/liquid.yml

```diff
@@ -6,7 +6,6 @@ jobs:
     strategy:
       matrix:
         entry:
-          - { ruby: '2.5', allowed-failure: false }
```

Comment:
This is needed because of ruby liquid dropping support for EOL ruby versions (https://github.com/Shopify/liquid/pull/1578)